crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -149,6 +163,11 @@ pub fn update_frusta<T: Component + CameraProjection + Send + Sync + 'static>(
     }
 }
 
+/// System updating the visibility of entities each frame.
+///
+/// The system is labelled with [`VisibilitySystems::CheckVisibility`]. Each frame, it updates the
+/// [`ComputedVisibility`] of all entities, and for each view also compute the [`VisibleEntities`]
```

Comment:
Too late, but ‘computes’

Reply:
Good catch, sorry I missed that. I'll try to make a new PR when I have some time.